generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.html.ejs

```diff
@@ -45,7 +45,10 @@
     </div>
     <%_ } _%>
     <br/>
-    <div class="table-responsive" *ngIf="<%=entityInstancePlural %>">
+    <div class="alert alert-warning" *ngIf="<%=entityInstancePlural %> && <%=entityInstancePlural %>.length === 0">
```

Comment:
I didn't used the totalItems field because it was not generated when using the no-pagination option, but it can be refactored to generate it every time

Reply:
You can replace `*ngIf="<%=entityInstancePlural %> && <%=entityInstancePlural %>.length === 0"`
With `*ngIf="<%=entityInstancePlural %>?.length === 0"

Reply:
Done